### PR TITLE
Adding warning about partially encrypted backup

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://encryption-backup-warning--ocpdocs.netlify.com/openshift-enterprise/latest/authentication/encrypting-etcd.html#enabling-etcd-encryption_encrypting-etcd

### CODE REVIEW

File: modules/enabling-etcd-encryption.adoc [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 You can enable etcd encryption to encrypt sensitive resources in your cluster.
 
-// [WARNING]
-// ====
-// Do not back up etcd until the encryption process is complete.
-// ====
+[WARNING]
+====
+It is not recommended to take a backup of etcd until the initial encryption process is complete, otherwise the backup might be only partially encrypted.
```
Comment (jboxman) [inline modules/enabling-etcd-encryption.adoc]: Could you say: _If you backup the etcd database before the initial encryption process is complete, then the backup might be only partially encrypted._ Or similar?

**Review by jboxman — APPROVED** [review]
One thought, otherwise LGTM.

File: modules/enabling-etcd-encryption.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/enabling-etcd-encryption.adoc]: @jboxman possibly, but I'd like to work in that it's not recommended to do so. Thoughts?

Comment (jboxman) [reply modules/enabling-etcd-encryption.adoc]: @bergerhoffer, possibly:

> It is not recommended to take a backup of the etcd database until the initial encryption process is complete. If the encryption process is not completed, the backup might be only partially encrypted.

Comment (bergerhoffer) [reply modules/enabling-etcd-encryption.adoc]: +1 thanks @jboxman. I do think that breaking it up into two sentences will be better for translation.

**Comment by bergerhoffer** [comment]
@geliu2016 Can you take a look and make sure that the wording around this warning is accurate? Thanks!

**Comment by geliu2016** [comment]
LGTM, thx

**Pull request merged**